### PR TITLE
only increment participant version after updates

### PR DESCRIPTION
**Pull request by paulwe**

### CODE REVIEW

File: pkg/rtc/uptrackmanager_test.go [hunk 1]
```diff
@@ -257,7 +257,7 @@ func TestSubscriptionPermission(t *testing.T) {
 				},
 			},
 		}
-		um.UpdateSubscriptionPermission(subscriptionPermission, nil, nil, nil)
+		um.UpdateSubscriptionPermission(subscriptionPermission, utils.TimedVersion{}, nil, nil)
```
Comment (boks1971) [inline pkg/rtc/uptrackmanager_test.go:260]: guess I was lazy to add tests for skipping older version when I initially added the subscription permission version 😞 . Mind adding some tests for the timed version check paths.

**Review by boks1971 — APPROVED** [review]
@paulwe LGTM! But, have this feeling that I am missing something. Can you please wait for 👀  from @davidzhao just to be sure?

File: pkg/rtc/uptrackmanager_test.go (hunk 1 above)
Comment (paulwe) [reply pkg/rtc/uptrackmanager_test.go:260]: we defer to syncstore for version checking. emitted values are ordered and all of the updates are processed in a single goroutine so we don't need to check them in implementation code anymore.

Comment (boks1971) [reply pkg/rtc/uptrackmanager_test.go:260]: We should probably add tests or remove any version check completely from the code. 'SyncStore' is an implementation, but this code is released independent of 'SyncStore'. I think it is fine to say this module does not do any version check (looked like it still checks although I might have misread the diff) and it is an external responsibility to provide the right version. But if this module is doing version checks, would be good to add some tests. Not necessary to do now. Can be added later.

**Review by davidzhao — APPROVED** [review]
lgtm!

should we start transitioning the simple version we are using to TimedVersion across the board in the proto itself?

**Comment by boks1971** [comment]
I vote for moving to including timed version uniformly everywhere

**Comment by paulwe** [comment]
yes, as long as we have good int64 support everywhere it would be great to have one consistent version format

**Pull request merged**